### DISCUSSION TITLE
Can't run in ChromeOS

### DISCUSSION THREAD

**Discussion by akra3523** [rustdesk/rustdesk#10369] [General]
### Bug Description

When I installed rustdesk-1.3.6-aarch64.deb and attempted to run it, the console displayed the following messages:

```
Cannot load libcuda.so.1
(rustdesk:3089): Gdk-CRITICAL **: 20:56:46.650: gdk_gl_context_make_current: assertion 'GDK_IS_GL_CONTEXT (context)' failed

** (rustdesk:3089): WARNING **: 20:56:46.650: Failed to initialize GLArea: Unable to create a GL context
```

The application did not show any interface. When I touched the screen, the console output the following error:

`** (rustdesk:3089): CRITICAL **: 20:56:47.497: void fl_keyboard_manager_sync_modifier_if_needed(FlKeyboardManager *, guint, double): assertion 'FL_IS_KEYBOARD_MANAGER(self)' failed`

When I manually terminated RustDesk, the terminal displayed this warning again:

`** (rustdesk:3089): WARNING **: 20:56:46.650: Failed to initialize GLArea: Unable to create a GL context`

Please help me resolve this issue and explain the potential cause.

### How to Reproduce

1.install deb
2.touch the app and run

### Expected Behavior

This App can be run normally

### Operating system(s) on local (controlling) side and remote (controlled) side

 Debian 12.8 aarch64 with ChromeOS 130

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.36

### Screenshots

![Screenshot 2024-12-25 21 58 00](https://github.com/user-attachments/assets/13cc8b6b-55b9-40c9-b60e-fd6abaaaee2b)

**Comment by xlionjuan** [comment]
Try https://flathub.org/apps/com.rustdesk.RustDesk
